custom_components/tuya_local/devices/afyeev_evcharger.yaml

```diff
@@ -42,6 +42,7 @@ secondary_entities:
   - entity: sensor
     class: energy
     category: diagnostic
+    optional: true
```

Comment:
It is the dp that needs to be marked optional, not the entity.

Reply:
ok, even with that I cannot make it work now. I have to understand what have been changed and what is blocking on my side.